survey-of-agency-tools.md

```diff
@@ -0,0 +1,149 @@
+# Survey of agency performance tools
+
+_Non-authoritative performance.gov research, curated by 18F, a digital
+consultancy in the US federal government._
+
+Agencies use many different software packages for tracking and collecting
+performance data. The tools listed here were included by 16 CFO agencies as part
+of a [survey](https://drive.google.com/drive/folders/0B2stDO5hLAHkZUxPYy04X3F3dk0)
+for performance.gov.
+
+We can't guarantee the accuracy of this page–it represents initial research
+of team members, and could be incomplete or become out-of-date.
+
+
+## Performance tools
```

Comment:
Maybe expand this to something like, "Performance measurement tools currently in use by surveyed agencies" for increased scannability.

Reply:
I might just remove this heading. Originally I was going to have two second level headings, one about the tools, and another about how the tools are being used, but I don't have any content for the second section so I dropped it (for now).